Resources/Prototypes/Entities/Clothing/Shoes/speedboots.yml

```diff
@@ -0,0 +1,37 @@
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingShoesSpeedBoots
+  name: speed boots
+  description: Allows you to run much faster!
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/speedboots.rsi
+    state: icon
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/speedboots.rsi
+  - type: StaticPrice
+    price: 2000
+  - type: ClothingSpeedModifier
+    walkModifier: 1
+    sprintModifier: 1.2
+    enabled: true
```

Comment:
Make it take power, should just need PowerCellDrawComponent + PowerCellComponent (I think I made a parent you can use for this) and a generic component for it.